pkg/bloombuild/builder/builder_test.go

```diff
@@ -149,14 +192,19 @@ func (f *fakePlannerServer) BuilderLoop(srv protos.PlannerForBuilder_BuilderLoop
 		if _, err := srv.Recv(); err != nil {
 			return fmt.Errorf("failed to receive task response: %w", err)
 		}
-		f.completedTasks.Add(1)
+		f.completedTasks.Inc()
+		time.Sleep(10 * time.Millisecond) // Simulate task processing time to add some latency.
```

Comment:
Should the sleep be before the task is counted as completed?

Reply:
Not necessarilly but I changed that.